### ISSUE TITLE
Feedback for failing minimum word count inadequate

### ISSUE THREAD

**Issue by Alex-Yarkosky** [GatorEducator/gatorgrader#138]
When `--words` is used in `gatorgrader.yml` to require a minimum number of words is met in the paragraphs detected, the current feedback only tells the user of the smallest word count found in all of the paragraphs it checked. It would be much more useful to the user if instead it told how many words it found in each of the paragraphs. This would allow the user to keep track of their progress if they jump around between the paragraphs they write and are trying to meet a minimum set in this way. It will also help locate where the minimum is not being met such as in the case provided below.

Sample output: 
```
-~-  FAILURES  -~-

✘  The reflection.md in writing has at least 100 word(s) in every paragraph
   ➔  Found 99 word(s) in a paragraph of the specified file
```

**Comment by gkapfham** [comment]
Hello @Alex-Yarkosky thanks for raising this issue. One of our design goals is to ensure that the diagnostic message is always a single line. What would you suggest we do if GatorGrader is checking a document that contains a lot of technical writing with many paragraphs? How do you think it would be best to maximize the use of the single line that we have to give feedback to the user of the tool?

**Comment by Alex-Yarkosky** [comment]
@gkapfham if the message should be contained within one line, one idea to consider would to only report on the paragraphs with less words than required giving the user the number of the paragraph and how many words it detected. An example of how this might look would be:
```
✘ The FILE_NAME in LOCATION has at least MIN_NUM_OF_WORDS word(s) in every paragraph
   ➔  These paragraphs need more words: (P#:PARAGRAPH_NUM, Ws:NUM_OF_WORDS), (P#:PARAGRAPH_NUM, Ws:NUM_OF_WORDS), (P#:PARAGRAPH_NUM, Ws:NUM_OF_WORDS), (P#:PARAGRAPH_NUM, Ws:NUM_OF_WORDS)
```
An example of the same message with sample numbers in it would look like the following:
```
✘ The FILE_NAME in LOCATION has at least 100 word(s) in every paragraph
   ➔  These paragraphs need more words: (P#:2, Ws:34), (P#:6, Ws:81), (P#:8, Ws:19), (P#:9, Ws:94)
```
This has the possibility of being longer than desired since the tool obviously cannot control how many paragraphs exist and happen to be under the required minimum number of words. Another possible idea would be to allow the user to provide an argument when `gradle grade` is run that would enable a more detailed report that can go over one line per diagnosis message. Let me know if there are any questions surrounding these ideas or if they would not be feasible. Feel free to also either explain why having all diagnosis messages be single line or point me in the direction of where I can find out more about this decision. Thanks!

**Comment by Michionlion** [comment]
It may be enough to simply show which paragraph number has the minimum number of words, and what that minimum is.

**Comment by Alex-Yarkosky** [comment]
@Michionlion that would be an appreciated improvement, what I was suggesting is just a step beyond that in order to provide the user as much information as possible as I see doing so of use

**Comment by gkapfham** [comment]
Hello @Michionlion and @Alex-Yarkosky, thanks for discussing this issue. I will try to work on it now and produce an improved implementation that provides some additional diagnostics.

**Comment by gkapfham** [comment]
This issue will be resolved once PR #171 is merged.
